hera_cal/io.py

```diff
@@ -2665,3 +2960,100 @@ def throw_away_flagged_ants_parser():
                     help="Also throw away baselines that have all channels and integrations flagged.")
     ap.add_argument("--clobber", default=False, action="store_true", help='overwrites existing file at outfile')
     return ap
+
+def uvdata_from_fastuvh5(
```

Comment:
Lots of un-covered code in this function... also, feel's like we've got a lot of duplicated functionality between these three functions... Can we condense and re-use code better?

Reply:
This is the function that is now actually used. I'll try make sure I cover it.